docs/javascripts/downloads.js

```diff
@@ -3,6 +3,84 @@
 
   if (!downloadsTableIdentifier) { return; }
 
+  const cellTitle = (version, date) => `
+    <div class="title">
+      <span class="title__version">
+        ${version}
+      </span>
+      <span class="title__release-date">
+        (${date})
+      </span>
+    </div>
+  `;
+  const apiLink = (name, version, prevVersion, visibility) => `
+    <div class="api-link">
+      <a target="_blank" href="https://api-insights.qlik.com/#/api-changes/core/${name}/${prevVersion}/${version}">
+        ${name}
+      </a>
+    </div>
+  `;
+  const circleProperties = (name) => {
+    switch (name.toLowerCase()) {
+      case 'added':
+        return { extraClass: 'changes__circle--added', title: 'Added' };
```

Comment:
Perhaps clarify `API methods changed`? Not everyone will know what the bubbles mean :)